src/serde/ser.rs

```diff
@@ -65,6 +65,13 @@ where
     Ok(serializer.output)
 }
 
+pub fn to_vec_unnamed<T>(value: &T) -> Result<Vec<u8>>
+where
+    T: Serialize,
+{
+    to_bytes_unnamed(value).map(|bytes| bytes.to_vec())
```

Comment:
I think you can replace the closure with a function reference, which looks nicer.